tsconfig.json

```diff
@@ -13,7 +13,16 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "baseUrl": "./",
+    "paths": {
+      "@lib/*": ["lib/*"],
+      "@components/*": ["components/*"],
+      "@stores/*": ["stores/*"],
```

Comment:
With `baseURL` specified to `./`, these shortcuts are actually the same as just using the actual path. `@lib/*` does the same thing as `lib/*`. So I'm not sure we need the shortcut with the `@`. Thoughts?

Reply:
have you tried it without the '@' ? i don't know if it works

Reply:
Using `lib/* works the same as `@lib/*`. I just made sure. Adding `"baseUrl": "./"` in `tsconfig.json` makes it work.

Reply:
ok, so we replace all the @ stuff